JS-Algorithms/challenges.js

```diff
@@ -118,7 +137,17 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+   let n1 = 0,
+      n2 = 1,
+      nextTerm = n1 + n2;
```

Comment:
Here you can improve the readability of your code by using let for each different variable